src/css/_uswds_variables.scss

```diff
@@ -0,0 +1,153 @@
+// These variables drive almost every aspect of the U.S. Web Design
+// system. Every variable is declared as !default so that it can be
+// overridden by a previous import.
+
+// Typography
+// Removing the !default from $em-base so we are not inheriting that
+// value from Bourbon.
+
+@import "../../node_modules/uswds/src/stylesheets/lib/addons/font-stacks";
```

Comment:
Had to add this line to pull in variable references to `$georgia` and `$helvetica`